projects.yml

```diff
@@ -5442,3 +5460,345 @@ truegrit:
   url: https://github.com/KingMob/TrueGrit
   categories: [Retrying Failures]
   platforms: [clj]
+
+ambly:
+  name: Ambly
+  url: https://github.com/mfikes/ambly
+  categories: [REPLs]
+  platforms: [cljs]
+
+pyramid:
+  name: Lacina
+  url: https://github.com/lilactown/pyramid
+  categories: [Graphs, EQL]
+  platforms: [clj, cljs]
+
+asami:
+  name: Asami
+  url: https://github.com/threatgrid/asami
```

Comment:
Active development of Asami is now at https://github.com/quoll/asami as noted in the [project readme](https://github.com/quoll/asami#asami--).

Reply:
Good catch, thanks.